front/components/assistant/conversation/input_bar/InputBar.tsx

```diff
@@ -99,7 +101,23 @@ export function AssistantInputBar({
   const sendNotification = useContext(SendNotificationsContext);
 
   const [isAnimating, setIsAnimating] = useState<boolean>(false);
-  const { animate, selectedAssistant } = useContext(InputBarContext);
+  const {
```

Comment:
Seen IRL. In practice, this is not the correct way to use React context. SWR is our component responsible for data handling. I recommend either of the following approaches:
- Move this functionality up one layer and add a `mentions` parameter to this component.
- Use this hook in the new screen with the same parameters to leverage SWR's capabilities.